From 317c680e96ac90bf44414bf779734353e5bd8c17 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 12 Feb 2020 12:15:27 -0600
Subject: [PATCH 081/151] firmware: intel_stratix10_service: add depend on
 agilex

commit  3460c8207f9824c2816cda8bf2d6bb53efa31967 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

[upstream commit 7db6e728c77f0ee3ca46dc8c62cf65eb412d9a2b]

Add depend on Agilex for Intel Agilex SoC platform.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 drivers/firmware/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/firmware/Kconfig b/drivers/firmware/Kconfig
index ab7f1e3bab09..aa42d366bd76 100644
--- a/drivers/firmware/Kconfig
+++ b/drivers/firmware/Kconfig
@@ -206,7 +206,7 @@ config FW_CFG_SYSFS_CMDLINE
 
 config INTEL_STRATIX10_SERVICE
 	tristate "Intel Stratix10 Service Layer"
-	depends on ARCH_STRATIX10 && HAVE_ARM_SMCCC
+	depends on (ARCH_STRATIX10 || ARCH_AGILEX) && HAVE_ARM_SMCCC
 	default n
 	help
 	  Intel Stratix10 service layer runs at privileged exception level,
-- 
2.26.1

